addons/bus/models/bus_presence.py

```diff
@@ -25,14 +26,14 @@ class BusPresence(models.Model):
     _description = 'User Presence'
     _log_access = False
 
-    user_id = fields.Many2one('res.users', 'Users', ondelete='cascade')
+    user_id = fields.Many2one(
+        'res.users', 'Users', ondelete='cascade',
+        index=unique(nulls=Nulls.DISTINCT, where="{field} is not null")
```

Comment:
Why do you need to specify `nulls=Nulls.DISTINCT` and `"{field} is not null"`, isn't it the default?